q2cli/tools.py

```diff
@@ -19,10 +19,10 @@ def tools():
 
 
 @tools.command(name='export',
-               short_help='Export data from a QIIME 2 Artifact'
+               short_help='Export data from a QIIME 2 Artifact '
                'or a Visualization',
-               help='Exporting extracts (and optionally transforms) data'
-               'stored inside an Artifact or Visualization. Note that'
+               help='Exporting extracts (and optionally transforms) data '
+               'stored inside an Artifact or Visualization. Note that '
                'Visualizations cannot be transformed with --output-format'
                )
 @q2cli.option('--input-path', required=True,
```

Comment:
Can we add a description to this option?

How about we match the `input-path` text in `import`?

`Path to file that should be exported`